hera_mc/correlator.py

```diff
@@ -653,27 +653,57 @@ class AntennaStatus(MCDeclarativeBase):
-    adc_mean: Mean ADC value, in ADC units (Float)
-    adc_rms: RMS ADC value, in ADC units (Float)
-    adc_power: Mean ADC power, in ADC units squared (Float)
-    pam_atten: PAM attenuation setting for this antenna, in dB. (Integer)
-    pam_power: PAM power sensor reading for this antenna, in dBm (Float)
-    pam_voltage: PAM voltage sensor reading for this antenna, in Volts (Float)
-    pam_current: PAM current sensor reading for this antenna, in Amps (Float)
-    pam_id: serial number of this PAM (string)
-    fem_voltage: FEM voltage sensor reading for this antenna, in Volts (Float)
-    fem_current: FEM current sensor reading for this antenna, in Amps (Float)
-    fem_id: serial number of this FEM (string)
-    fem_temp: EM temperature sensor reading for this antenna in degrees Celsius (Float)
-    eq_coeffs: Digital EQ coefficients for this antenna, list of floats stored as a string (String)
-    histogram_bin_centers: ADC histogram bin centers, list of ints stored as a string (String)
-    histogram: ADC histogram counts, list of ints stored as a string (String)
+    Attributes
+    ----------
+    time : BigInteger
+        gps time of the snap status data, floored.
+        part of primary_key
+    antenna_number : Integer
+        antenna number
+        part of primary_key
+    antenna_feed_pol : String
+        Feed polarization, either 'e' or 'n'
+        part of primary_key
+    snap_hostname : String
+        snap hostname
+    snap_channel_number : Integer
+        The SNAP ADC channel number (0-7) to which this antenna is connected
+    adc_mean : Float
+        Mean ADC value, in ADC units
+    adc_rms : Float
+        RMS ADC value, in ADC units
+    adc_power : Float
+        Mean ADC power, in ADC units squared
+    pam_atten : Integer
+        PAM attenuation setting for this antenna, in dB.
+    pam_power : Float
+        PAM power sensor reading for this antenna, in dBm
+    pam_voltage : Float
+        PAM voltage sensor reading for this antenna, in Volts
+    pam_current : Float
+        PAM current sensor reading for this antenna, in Amps
+    pam_id : String
+        serial number of this PAM (string)
+    fem_voltage : Float
+        FEM voltage sensor reading for this antenna, in Volts
+    fem_current : Float
+        FEM current sensor reading for this antenna, in Amps
+    fem_id : String
+        serial number of this FEM
+    fem_temp : float
+        EM temperature sensor reading for this antenna in degrees Celsius
+    eq_coeffs : String
+        Digital EQ coefficients for this antenna, list of floats stored as a string
+    histogram_bin_centers : String
+        ADC histogram bin centers, list of ints stored as a string
+    histogram : String
+        ADC histogram counts, list of ints stored as a string
+    autocorrelation : String
+        Real valued autocorrelation value for the given antenna pol
+        list of floats stored as string
```

Comment:
Wow, thanks!

Reply:
How long are these lists generally?

Reply:
Truth be told I am not 100% sure, if they are the same length as the 'autos' from the redis database they will be `int(2048 // 4 * 3) = 1536` entries long.

Reply:
ok. I think we'll just need to keep an eye on the database size. We might need to archive older values and remove them from the onsite db if it gets too big.